service/policy/db/migrations/20240402000000_preserve_value_order.sql

```diff
@@ -36,13 +36,21 @@ EXECUTE FUNCTION update_definition_delete_values_order();
 -- +goose Down
 
 -- +goose StatementBegin
+DROP TRIGGER IF EXISTS trigger_update_definition_add_values_order ON attribute_values;
+-- +goose StatementEnd
```

Comment:
For my learning, do we need to mark these as different statements within the down migration?